src/lib.rs

```diff
@@ -64,6 +87,9 @@
 //! by using [`from_slice`] or by accessing the underlying raw object of a [`Bmp`] object with
 //! [`Bmp::as_raw`].
 //!
+//! Similar to [`Bmp::pixel`], [`RawBmp::pixel`] can be used to get raw pixel color values as a
```

Comment:
`Bmp::pixel` and `RawBmp::pixel` will need to be added to the `README-LINKS`. (see failing CI)